### PR TITLE
Test all L2 converter verbs

### PR DESCRIPTION
**Pull request by JFernandezAdobe**
Jira ticket: [MWPW-126088](https://jira.corp.adobe.com/browse/MWPW-126088)

Test will cover the following E2E conversion flow:

1. Upload a test document
2. Wait for conversion to complete
3. Wait for file preview
4. Download the converted file

Test pages:
https://main--dc--adobecom.hlx.page/acrobat/online/pdf-to-ppt
https://main--dc--adobecom.hlx.page/acrobat/online/pdf-to-jpg
https://main--dc--adobecom.hlx.page/acrobat/online/pdf-to-word
https://main--dc--adobecom.hlx.page/acrobat/online/pdf-to-excel
https://main--dc--adobecom.hlx.page/acrobat/online/convert-pdf
https://main--dc--adobecom.hlx.page/acrobat/online/ppt-to-pdf
https://main--dc--adobecom.hlx.page/acrobat/online/jpg-to-pdf
https://main--dc--adobecom.hlx.page/acrobat/online/word-to-pdf
https://main--dc--adobecom.hlx.page/acrobat/online/excel-to-pdf

### CODE REVIEW

**Comment by JFernandezAdobe** [comment]
Hi @amauch-adobe, FYI this PR requires that the previous PR be merged first for the tests to pass.
Thanks!
https://github.com/adobecom/nala/pull/61

File: features/dc_converter.spec.js [hunk 1]
```diff
@@ -5,6 +5,14 @@ module.exports = {
...
+        '/acrobat/online/pdf-to-jpg',
+        '/acrobat/online/pdf-to-word',
+        '/acrobat/online/pdf-to-excel',
+        '/acrobat/online/convert-pdf',
+        '/acrobat/online/ppt-to-pdf',
+        '/acrobat/online/jpg-to-pdf',
+        '/acrobat/online/word-to-pdf',
+        '/acrobat/online/excel-to-pdf',
       ],
       envs: '@dc_preview @dc_prod',
```
Comment (amauch-adobe) [inline features/dc_converter.spec.js:17]: See my previous comments in PR #61 about how you can separate URLs that require the .HTML extension in your spec file.  The Parse.js file shouldn't have specific implementation outside of the normal core Franklin/Milo support.  Having the .HTML extension isn't Milo/Franklin's best practice but required support for legacy compatibility.  It should remain either in Specs or test implementations and not core Nala.

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
Looks good, my only request is to move your data docs down a directory, meaning move them to /docs/dc.  That way we can separate other teams' docs and organize things for the time being, and then in the future make it easier to pull out teams' code/files to their separate repos.

**Review by Dli3 — APPROVED** [review]

**Review by amauch-adobe — APPROVED** [review]
Approved.

**Pull request merged**